src/player.js

```diff
@@ -202,6 +202,14 @@ const MPRISPlayer = new Lang.Class({
           if (props.CanGoPrevious)
             newState.canGoPrevious = props.CanGoPrevious.unpack();
 
+          if (props.CanSeek) {
+            newState.canSeek = props.CanSeek.unpack();
+            if (this.state.canSeek !== newState.canSeek) {
+              this.emit('player-update', newState);
```

Comment:
I agree with the checking 1st bit. But if you don't emit 'player-update' before you call _getPosition won't it have stale state.canSeek info?